docs/sources/installation/configuration.md

```diff
@@ -383,37 +383,6 @@ below.
 - [LDAP Authentication]({{< relref "auth/ldap.md" >}}) (auth.ldap)
 - [Auth Proxy]({{< relref "auth/auth-proxy.md" >}}) (auth.proxy)
 
-## [session]
```

Comment:
Not completely sure what to do here - either remove it or mark it as `removed in ...`  note

Reply:
Good question. I dont think we ever removed configuration settings this large before. 
I'm leaning towards keeping all of it and adding a note about it being removed in 6.1.

@torkelo @marefr @xlson whats your take on this?

Reply:
Do we have any kind of design to mark it?

For example at jquery we do something like this - https://api.jquery.com/andSelf/
Notice:
> version deprecated: 1.8, removed: 3.0

What should we do here? Since it seems we wouldn't want users to continue using this option on older versions of grafana?

Reply:
Moved it to the end of the page with above title `Removed options`. Similar how it was done over here - https://grafana.com/docs/http_api/dashboard/#deprecated-resources